x-pack/plugins/alerts/server/task_runner/create_execution_handler.ts

```diff
@@ -39,6 +39,7 @@ export interface CreateExecutionHandlerOptions<
   actions: AlertAction[];
   spaceId: string;
   apiKey: RawAlert['apiKey'];
+  kibanaBaseUrl: string | undefined;
```

Comment:
there were a couple of paths of threading this new variable into the right place, this required the fewest number of changes